azure/templates/tests.yml

```diff
@@ -27,6 +27,7 @@ steps:
           -e REASONABLE_ADJUSTMENTS_BASE_URL=https://$(APIGEE_ENVIRONMENT).api.service.nhs.uk \
           -e REASONABLE_ADJUSTMENTS_PROXY_PATH=$(SERVICE_BASE_PATH) \
           -e REASONABLE_ADJUSTMENTS_PROXY_NAME=$(FULLY_QUALIFIED_SERVICE_NAME) \
+          -e STATUS_APIKEY="b3515bf1-d0a8-482f-8785-a5c0fdc19a52" \
```

Comment:
Are you sure we want this be hardcoded here? Maybe we need to add this to secret manager?

Reply:
spoke to Paul Walkington and he deemed this apikey non-sensitive